cg/meta/workflow/nf_handlers.py

```diff
@@ -21,11 +21,13 @@ class NfBaseHandler:
     def write_nextflow_yaml(
         cls,
         content: Dict[str, Any],
-        file_path: str,
+        file_path: Path,
     ) -> None:
         """Write Nextflow file with non-quoted booleans and quoted strings."""
         with open(file_path, "w") as outfile:
             for key, value in content.items():
+                if isinstance(value, Path):
```

Comment:
Could you rename `key` and `values` to something more specific to this case? 👼

Reply:
I cannot come up with anything better than key and value. Depending on the content to write the keys could be parameters ids or something else, but this function is quite general. It writes a yaml block that works with nextflow. Unless you have a specific suggestion I think I will leave it like that

Reply:
I see, then I would say it's ok if it's not possible to provide accurate information 👍